### DISCUSSION TITLE
Grafana seperate RootFS and NSM stats

### DISCUSSION THREAD

**Discussion by DebianGuru** [Security-Onion-Solutions/securityonion#9674] [Unsupported Versions]
In Grafana, there's a dashboard for my standalone machine.  It contains all the various graphs/dials/etc for hardware usage.  The two that I'm looking at are labeled "RootFS used" and "NSM used".  Apparently this was configured for people such as myself who have a seperate /NSM to store data.  However, the RootFS is looking at "/", which of course will INCLUDE the /NSM folder (where my data partition is mounted).  Therefore, it always shows that my RootFS is nearly full (which in fact it's only about 45% full).

How can I change this so RootFS EXCLUDES the /NSM folder?  Thanks.

**Comment by InfosecGoon** [comment]
It should exclude that by default, if you're using a separate disk partition for /nsm.

Can you share the output from "df -h" and "lsblk"?

Also, what version of Security Onion are you using (/etc/soversion) and is it running on CentOS or Ubuntu?

**Reply by DebianGuru** [reply]
df -h
Filesystem                              Size  Used Avail Use% Mounted on
udev                                     45G     0   45G   0% /dev
tmpfs                                   8.9G  3.6M  8.9G   1% /run
/dev/mapper/ubuntu--vg-ubuntu--lv        93G   46G   44G  51% /
tmpfs                                    45G  944K   45G   1% /dev/shm
tmpfs                                   5.0M     0  5.0M   0% /run/lock
tmpfs                                    45G     0   45G   0% /sys/fs/cgroup
/dev/mapper/ubuntu--vg-ubuntu--lv--nsm  5.1T  4.3T  565G  89% /nsm
/dev/sdc2                               3.6T  2.9T  588G  84% /mnt/usb
/dev/loop2                               64M   64M     0 100% /snap/core20/1738
/dev/loop1                               56M   56M     0 100% /snap/core18/2667
/dev/loop3                               64M   64M     0 100% /snap/core20/1778
/dev/loop4                               50M   50M     0 100% /snap/snapd/17883
/dev/loop5                               92M   92M     0 100% /snap/lxd/23991
/dev/loop6                               92M   92M     0 100% /snap/lxd/24061
/dev/sda2                               974M  205M  702M  23% /boot
//data/deptdocs/users/jayf              3.6T  750G  2.8T  21% /home/jayf/smb/jayf
//devdta/x$                             1.8T   70G  1.7T   4% /home/jayf/smb/devdta_x
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/45247b94305e81a014c37ab47d49bb026ddd68f2e070ec6e5e260e7403ff365e/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/c3f51aef8d36743842af1f6badc84929de94dcf5fb9a47ca8aaca35e000dc454/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/e15e0edf02ead8ea0166c54c69cd7d4a86e894ea1270ee3cef7bcf27a21b8bb7/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/ff48821c13d9360b01b0258b455cca99d820119d8f95bf33371b899f199c5522/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/8d1555763ee7eabee09c9faf259a935aafec201a01d0ee568cf32f8a0feaac99/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/9dae6c466f1e42cf7e47793dfb451324113da10d3b0d3bf89a0ab010d07db9fa/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/f7084c1917c1b559dfd058f38fba5ec98eb745442e04441bb926c706d3a4ca5b/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/5779cc81807deb9ae3ee6a7ca08e4acd494bffce5eed17dcd0c8f2c3374893c6/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/aa4ba6e672f376c0e72265ac7e6c1ed25d628ded480c0239260e1dd559868187/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/d7b0c7766ebb078008eede393852ec769756ff31c4e0b773e8e044472b71369b/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/bf23d7db1f2aba237491b1597b1e80eba460ac73e4517bde0c4c311e0d1e7dfa/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/e6a51c5464d354db55899e7f36afeece90756d2d5ab16b2b014cc6c8a52bfeaf/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/ee761e4c6e5ac937fa4afb4617ccb637616393a329004013962ef36387692a3d/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/abe93b468d38685debf8c7498e72133984274e9538344464f8586e16c7656b94/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/dcf0a9be317ab3f4edc66ac05a6d3e47b5e8242b82cec475398e076a9aff80bb/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/f50fd050c417e3f7aaa3e7173abb5dd372481b3e89d7160e4063e006e560f63f/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/749f56a0a473580cfd6c6abee83573e8695452cf425e2f2aee6df95106ea568f/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/6d1c0a968804f19163ab846f7c7f6e168ef746eece9bba2e65b3006a00625695/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/5b5da0e28c171cc792c2fc0e01a0714407a5b8d50db9e261e9784b21eb3bf21c/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/cfe4fe7af77b54e5566a5320a2deddf06e6b891fcd50f27f75658c9ae587490b/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/7efad5323358edfd6d6ac6edf8a3a58f6ef4bbe120ca616e5b2ac2c93fa8f8a9/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/ee629171111065ce8d632173cf59672449b03b555a5e0883041c36c9e30c0510/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/6dfbd60f4f01937e4337442378a04d03ce70b23fa7d778de0bccf43dc47f990b/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/36ce5bcdedb5235835ffa0100489c38b6eb2a156928740c5ff81ff0c5a57969a/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/dc9f9324cc81f3cfcb73ec7a62b51837bea6c498687d50fcd772749f6e3db3d4/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/c863a4afba5767da0300ffbb83dc66722fd74f9a6bb6bef3c92f44858d855c4b/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/a9d8966515fca825056946431e3ec3adb6e8f73fcd26375af212f74226771ebb/merged
/dev/loop7                               50M   50M     0 100% /snap/snapd/17950
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/128930b06a251de7eb202c6f2a9acd5a2650e689464adddacff12823b2350d21/merged
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/e0c7aa7e8f28d26319c67f4f11dabe48a12a6f4c2a2159e96b865eee103b5b88/merged
tmpfs                                   8.9G     0  8.9G   0% /run/user/1000
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/9b511a149096f49a3a90769c511c0d122d5aa0722b17e2de67b5e86de3435994/merged
/dev/loop8                               56M   56M     0 100% /snap/core18/2679
overlay                                 5.1T  4.3T  565G  89% /nsm/docker/overlay2/954e0d009dc36b10514f3e48ea31df71322e158dda809e9a14fc9568d130758d/merged

lsblk
NAME                           MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
loop1                            7:1    0 55.6M  1 loop /snap/core18/2667
loop2                            7:2    0 63.2M  1 loop /snap/core20/1738
loop3                            7:3    0 63.3M  1 loop /snap/core20/1778
loop4                            7:4    0 49.6M  1 loop /snap/snapd/17883
loop5                            7:5    0 91.8M  1 loop /snap/lxd/23991
loop6                            7:6    0 91.9M  1 loop /snap/lxd/24061
loop7                            7:7    0 49.8M  1 loop /snap/snapd/17950
loop8                            7:8    0 55.6M  1 loop /snap/core18/2679
sda                              8:0    0  5.2T  0 disk
├─sda1                           8:1    0    1M  0 part
├─sda2                           8:2    0    1G  0 part /boot
└─sda3                           8:3    0  5.2T  0 part
  ├─ubuntu--vg-ubuntu--lv      253:0    0   95G  0 lvm  /
  └─ubuntu--vg-ubuntu--lv--nsm 253:1    0  5.1T  0 lvm  /nsm
sdb                              8:16   0   20G  0 disk
└─sdb1                           8:17   0   20G  0 part
sdc                              8:32   0  3.7T  0 disk
├─sdc1                           8:33   0  512M  0 part
└─sdc2                           8:34   0  3.7T  0 part /mnt/usb
sr0                             11:0    1  396M  0 rom
